### ISSUE TITLE
Dismiss button text overflow in IE 11

### ISSUE THREAD

**Issue by webholics** [insites/cookieconsent#326]
The text in the dismiss button overflows in IE 11 if its longer than the set `min-width: 140px` without letting the button grow as expected.

**Comment by jaegerschnitzel** [comment]
Here is a screenshot of this bug: http://666kb.com/i/dtohjve0o0lklyi5y.png

**Comment by OldManBrook** [comment]
I used two buttons to get around this overflow issue in IE:
```
"showLink": true,
		"theme": "classic",
		"type": "opt-in",
		"content": {
		"message": "You can choose to deny or allow use of Cookies during your visit..",
			"dismiss": '<a aria-label="deny cookies" role="button" tabindex="0" class="cc-btn cc-deny" style="min-width: 140px;">Deny</a>',
			"allow": "Allow",
			"link": "Learn More",
			"href": "/contact/privacypolicy.html",
		},
```

**Comment by karlschwaier** [comment]
This issue seems to be caused by setting those buttons to flex: 1 in https://github.com/insites/cookieconsent/blob/7a493e8996faa691f47c777225ec66095fdd14d6/src/styles/layout.css#L55-L57.

I got around this with some additional CSS:

```
.cc-compliance > .cc-btn {
    flex: none !important;
}
```

**Comment by Luperi** [comment]
Also fixed with: 

`.cc-compliance > .cc-btn {
    flex: 1 0 0;
}`

**Comment by Dolondro** [comment]
This bug should be fixed in 3.1.0

**Comment by thexmanxyz** [comment]
@Dolondro Do confirm that the bug is fixed!
